### PR TITLE
Hopper agitate controls

### PR DESCRIPTION
**Pull request by NikhilSuresh24**
Added a button on the teensy stick to start the HopperAgitate command.

### CODE REVIEW

**Comment by billwpierce** [comment]
Please test.

**Comment by billwpierce** [comment]
@NikhilSuresh24 please test.

File: .classpath [hunk 1]
```diff
@@ -12,5 +12,7 @@
 	</classpathentry>
 	<classpathentry kind="var" path="opencv" sourcepath="opencv.sources"/>
 	<classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib-javadoc.jar"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/navx_frc-sources.jar"/>
```
Comment (ajnadel) [inline .classpath]: Dirty change stinky change take it away

File: src/org/usfirst/frc4904/robot/RobotMap.java [hunk 2]
```diff
@@ -162,7 +163,7 @@ public RobotMap() {
 		// Human inputs
 		Component.operatorStick = new CustomJoystick(Port.HumanInput.joystick);
 		Component.operatorStick.setDeadzone(HumanInterfaceConfig.JOYSTICK_DEADZONE);
-		Component.teensyStick = new CustomJoystick(Port.HumanInput.teensyStick);
+		Component.teensyStick = new TeensyController(Port.HumanInput.teensyStick, 30);
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: Move magic number to constant in human interface config..?

Comment (ajnadel) [reply src/org/usfirst/frc4904/robot/RobotMap.java]: Also will there really be 30 buttons?

File: .classpath [hunk 3]
```diff
@@ -12,5 +12,7 @@
 	</classpathentry>
 	<classpathentry kind="var" path="opencv" sourcepath="opencv.sources"/>
 	<classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib-javadoc.jar"/>
```
Comment (c0d3rman) [inline .classpath]: Revert classpath changes

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 4]
```diff
@@ -64,7 +65,8 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
 		RobotMap.Component.driverXbox.b.onlyWhileHeld(HumanInterfaceConfig.gearAlign);
 		RobotMap.Component.driverXbox.b.whenReleased(normalDrive);
-		RobotMap.Component.teensyStick.button1.whenPressed(normalDrive);
+		RobotMap.Component.teensyStick.getButton(9).whenPressed(new HopperAgitate());
```
Comment (c0d3rman) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: List in numerical order

**Review by c0d3rman — CHANGES_REQUESTED** [review]

File: src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java [hunk 5]
```diff
@@ -32,6 +33,7 @@ public void bindCommands() {
 			true)
 				.start();
 		RobotMap.Component.gearIO.setRampState(GearIO.RampState.EXTENDED);
+		RobotMap.Component.teensyStick.getButton(9).whenPressed(new HopperAgitate());
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java]: I think you mean `onlyWhileHeld`, right?

File: src/org/usfirst/frc4904/robot/RobotMap.java (hunk 2 above)
Comment (NikhilSuresh24) [reply src/org/usfirst/frc4904/robot/RobotMap.java]: just a placeholder

**Review by ajnadel — APPROVED** [review]

**Review by c0d3rman — APPROVED** [review]

**Review by c0d3rman — APPROVED** [review]

**Review by c0d3rman — APPROVED** [review]

**Pull request merged**